apps/custom/tests/memcheck/run_litmus_test.yml

```diff
@@ -28,7 +28,7 @@ spec:
             value: memleak
 
         command: ["/bin/bash"]
-        args: ["-c", "ansible-playbook ./memleak/tests/test.yml -i /etc/ansible/hosts -v; exit 0"]
+        args: ["-c", "ansible-playbook ./custom/tests/memcheck/test.yml -i /etc/ansible/hosts -v; exit 0"]
         volumeMounts:
```

Comment:
@nsathyaseelan Can we update the memory check litmus path in the associated gitlab script for the other platforms as well?